Classes/Views/Constants.swift

```diff
@@ -42,5 +42,6 @@ enum Constants {
         static let inbox = NSLocalizedString("Inbox", comment: "")
         static let upload = NSLocalizedString("Upload", comment: "")
         static let bookmark = NSLocalizedString("Bookmark", comment: "")
+        static let removeBookmark = NSLocalizedString("Remove Bookmark", comment: "")
```

Comment:
Is this used anywhere else?

Reply:
Nope. I just don't like to have random string in AlertActions

Reply:
Although I get it, I think it would be more in line to keep single-use strings "hardcoded" as to not clutter the Constants *even more*.